backend/Tournament/managers.py

```diff
@@ -99,6 +99,26 @@ def raise_error_if_inactive_user_in_tournament(tournament: Tournament) -> None:
         inactive_users = tournament_players.filter(user__is_active=False)
         if inactive_users.exists():
             raise TournamentInProgressException(f"An user in the unfinished tournament has deleted their account")
```

Comment:
A user*

Reply:
shouldn't it be 'an' though? the user starts with a vowel.